src/core/heating_cooling_loads.jl

```diff
@@ -62,18 +56,18 @@ struct DomesticHotWaterLoad
         annual_mmbtu::Union{Real, Nothing} = nothing,
         monthly_mmbtu::Array{<:Real,1} = Real[],
         # addressable_load_fraction,  # TODO
-        fuel_loads_mmbtu_per_hour::Array{<:Real,1} = Real[],  # TODO this should be per_time_step, issue created
-        time_steps_per_hour::Int = 1,
+        fuel_loads_mmbtu_per_time_step::Array{<:Real,1} = Real[],
+        time_steps_per_hour::Int = 1, # corresponding to `fuel_loads_mmbtu_per_time_step`
         latitude::Real=0.0,
         longitude::Real=0.0
     )
-        if length(fuel_loads_mmbtu_per_hour) > 0
+        if length(fuel_loads_mmbtu_per_time_step) > 0
 
-            if !(length(fuel_loads_mmbtu_per_hour) / time_steps_per_hour ≈ 8760)
-                error("Provided domestic hot water load does not match the time_steps_per_hour.")
+            if !(length(fuel_loads_mmbtu_per_time_step) / time_steps_per_hour ≈ 8760)
+                throw(@error "Provided domestic hot water load does not match the time_steps_per_hour.")
             end
 
-            loads_kw = fuel_loads_mmbtu_per_hour .* (KWH_PER_MMBTU * EXISTING_BOILER_EFFICIENCY)
+            loads_kw = fuel_loads_mmbtu_per_time_step .* (KWH_PER_MMBTU * EXISTING_BOILER_EFFICIENCY * time_steps_per_hour)
```

Comment:
@Bill-Becker could you review the addition of `* time_steps_per_hour` here?

Reply:
Undid this change in https://github.com/NREL/REopt.jl/pull/82/commits/ff447d6e42571aab42437d61c5b95a7a19c7e955